scenes/MentorsScene.js

```diff
@@ -1,18 +1,186 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, WebView, ScrollView, TextInput, TouchableWithoutFeedback, TouchableOpacity, StyleSheet } from 'react-native';
+
+var HEADER = '#1A2E33';
+var BGWASH = 'rgba(255,255,255,0.8)';
+var DISABLED_WASH = 'rgba(255,255,255,0.25)';
+
+var TEXT_INPUT_REF = 'urlInput';
+var WEBVIEW_REF = 'webview';
+var DEFAULT_URL = 'https://mentorship.bitca.mp';
 
 export default class MentorsScene extends Component {
-  static get defaultProps() {
-    return {
-      title: 'Mentors'
-    };
-  }
+  state = {
+    url: DEFAULT_URL,
+    status: 'No Page Loaded',
+    backButtonEnabled: false,
+    forwardButtonEnabled: false,
+    loading: true,
+    scalesPageToFit: true,
+  };
+
+  inputText = '';
+
+  handleTextInputChange = (event) => {
+    var url = event.nativeEvent.text;
+    if (!/^[a-zA-Z-_]+:/.test(url)) {
+      url = 'https://' + url;
+    }
+    this.inputText = url;
+  };
 
   render() {
+    this.inputText = this.state.url;
+
     return (
-      <View>
-        <Text>Hi! My name is {this.props.title}.</Text>
+      <View style={[styles.container]}>
+        <View style={[styles.addressBarRow]}>
+          <TouchableOpacity
+            onPress={this.goBack}
+            style={this.state.backButtonEnabled ? styles.navButton : styles.disabledButton}>
+            <Text>
+               {'<'}
+            </Text>
+          </TouchableOpacity>
+          <TouchableOpacity
+            onPress={this.goForward}
+            style={this.state.forwardButtonEnabled ? styles.navButton : styles.disabledButton}>
+            <Text>
+              {'>'}
+            </Text>
+          </TouchableOpacity>
+        </View>
+        <WebView
+          ref={WEBVIEW_REF}
```

Comment:
Use a callback function instead
https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute